src/registry_schemas/schemas/filing.json

```diff
@@ -70,7 +70,8 @@
                 "transition",
                 "registrarsNotation",
                 "registrarsOrder",
-                "putBackOn"
+                "putBackOn",
+                "adminFreeze"
```

Comment:
Could you put these in alphabetical order instead of appending them?
I know a couple of others snuck by.